### PR TITLE
Replace AUC with accuracy in docs

### PR DESCRIPTION
**Pull request by nzw0301**
## Motivation

The examples use accuracy as an objective, not AUC.

- https://github.com/optuna/optuna-examples/blob/07c77d8baa17e18ee0b85435df080c11749c6777/xgboost/xgboost_simple.py#L62
- https://github.com/optuna/optuna-examples/blob/07c77d8baa17e18ee0b85435df080c11749c6777/lightgbm/lightgbm_simple.py#L42

## Description of the changes

Replace `AUC` with `accuracy`.

### Dear reviewers

I suppose this PR can be merged into the master branch with one approval.

### CODE REVIEW

**Review by himkt — APPROVED** [review]

**Review by toshihikoyanase — APPROVED** [review]
Good catch! Thank you!
